### PR TITLE
(HDS-2137) Update new icons to icons list in site

### PR DESCRIPTION
**Pull request by mrTuomoK**
## Description

Update to icons to site icons list

## Related Issue

[HDS-2137](https://helsinkisolutionoffice.atlassian.net/browse/HDS-2137)

## How Has This Been Tested?

- locally built site

## Add to changelog
- [x] Added needed line to changelog 

[HDS-2137]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-2137?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +46,7 @@ Changes that are not related to specific components
 #### Fixed
 
 - [Component] What bugs/typos are fixed?
+- [Icons] document-group icon wrong colors
```
Comment (NikoHelle) [inline CHANGELOG.md:49]: Capitalise "document"

File: release/icon-kit-template-CHANGELOG.txt
Comment (laurakarhu) [inline release/icon-kit-template-CHANGELOG.txt:1]: Should be 3.6.0 as 3.5.0 has been released already?

**Review by laurakarhu — CHANGES_REQUESTED** [review]
Otherwise good, but wrong version number?

File: release/icon-kit-template-CHANGELOG.txt
Comment (mrTuomoK) [reply release/icon-kit-template-CHANGELOG.txt:1]: Nope, it's bumped as we do the next release. (this was accidentally left one version too low last time I guess 🤦 )

Comment (mrTuomoK) [reply release/icon-kit-template-CHANGELOG.txt:1]: But the new icon-kit -workflow should now handle this better and automatically so it "can't be left behind"

Comment (mrTuomoK) [reply release/icon-kit-template-CHANGELOG.txt:1]: ping @laurakarhu so all should be ok?

**Review by laurakarhu — APPROVED** [review]
Let's see how the script works in practice

**Pull request merged**